### DISCUSSION TITLE
Dumping Ganbare Goemon Gaiden 1/2

### DISCUSSION THREAD

**Discussion by benbini** [sanni/cartreader#247] [Need help]
I am having difficulty dumping ganbare goemon gaiden [1](https://nesdir.github.io/EB92B32A_Japan.html) and [2](https://nesdir.github.io/286FCD20_Japan.html) (mappers 25 and 21 respectively; 256/256 prg/chr for both).  Other Famicom games, including Ganbare Goemon 1/2, dump successfully, and the latter games even belong to the same "family" of mappers.   

Looking through the technical information on nesdir.github.io, I see that unlike Goemon 1/2, Goemon Gaiden 1/2 both have 8k WRAM, which I assume is "Write RAM" storage for save files.  Does "WRAM" correspond to the "RAM" option that Sanni prompts for during mapping configuration, and if so do I need to set it (to 8k in this case) to dump Gaiden 1/2 successfully, even though I don't want to retrieve any save file information that might be on the cartridge?

**Comment by sanni** [comment]
Yes that is correct. I should add that information to the Wiki.
